infrastructure_files/management.json.tmpl

```diff
@@ -29,9 +29,9 @@
     "Datadir": "",
     "HttpConfig": {
         "Address": "0.0.0.0:$NETBIRD_MGMT_API_PORT",
-        "AuthIssuer": "https://$NETBIRD_AUTH0_DOMAIN/",
-        "AuthAudience": "$NETBIRD_AUTH0_AUDIENCE",
-        "AuthKeysLocation": "https://$NETBIRD_AUTH0_DOMAIN/.well-known/jwks.json",
+        "AuthIssuer": "$NETBIRD_AUTH_AUTHORITY",
```

Comment:
There is a problem with AuthIssuer when using Auth0 (needs keycloak test). When we enter an AUTH_AUTHORITY without the trailing forward slash (/) our management is unable to validate the token.
```shell
AUTH_AUTHORITY=https://auth0-domain.com
```
One fix would be to keep the keep the trailing forwarding slash:
```shell
"AuthIssuer": "$NETBIRD_AUTH_AUTHORITY/",
```

Reply:
This property is user set, so it should be the full authority including /

Reply:
ok